tests/__init__.py

```diff
@@ -1 +1,6 @@
 #
+try:
```

Comment:
@brendan-ward, It wasn't my first choice, but you need to change the backend to agg before any import of mpl.pyplot is done.  I originally tried it in test_tool.py but it didn't work (the plots still blocked)  I'm assuming it was because that import had already happened somewhere else by the time that test was run.